### PR TITLE
Fix typos and remove extra whitespaces

### PR DESCRIPTION
**Pull request by chutified**
This PR fixes some typos and removes extra whitespaces at the end of lines.

### CODE REVIEW

File: content/concepts/protocols.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ match for the protocol id, the handler function will be invoked with the new str
 The second kind of protocol registration takes three arguments: the protocol id, a protocol match function, and the handler function.
 
 When a stream request comes in whose protocol id doesn't have any exact matches, the protocol id will be passed through
-all of the registered match functions. If any returns `true`, the associated handler function will be invoked.
+all the registered match functions. If any returns `true`, the associated handler function will be invoked.
```
Comment (marten-seemann) [inline content/concepts/protocols.md:103]: This is correct.

**Review by marten-seemann — CHANGES_REQUESTED** [review]
I don't think it's necessary to fix typos in the Hugo example site.

File: content/concepts/protocols.md (hunk 1 above)
Comment (chutified) [reply content/concepts/protocols.md]: Yup, I changed it back.

**Comment by chutified** [comment]
All changes made to the Hugo example site were rolled back.

**Review by marten-seemann — APPROVED** [review]

**Comment by marten-seemann** [comment]
Thank you!

**Pull request merged**